demos/interactive_face_detection_demo/cpp_gapi/main.cpp

```diff
@@ -39,6 +39,7 @@
 #include <opencv2/gapi/gstreaming.hpp>
 #include <opencv2/gapi/infer.hpp>
 #include <opencv2/gapi/infer/ie.hpp>
```

Comment:
Should ie.hpp be removed?

Reply:
Sure, done https://github.com/openvinotoolkit/open_model_zoo/pull/3889/commits/29f61fb94a09c26956378261f33ffad8fe4789bf. Thanks!